### PR TITLE
fix: removed invalidateOptionsMenu() from the onDrawerOpened() and on…

### LINKED ISSUE #1446: Menu icons disappearing, re-appear and disappear again
The menu icons have a strange behavior at times and disappear, sometimes re-appear and disappear again.

screenshot_2017-04-30-14-39-34

### PR DESCRIPTION
**Pull request by arp95**
…DrawerClosed() functions

Fixes #1446 
Changes: 

removed invalidateOptionsMenu() from the onDrawerOpen() and onDrawerClosed() functions to ensure menu icon disappearing doesnt happen.

Screenshots for the change: 

-

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/utils/SmoothActionBarDrawerToggle.java [hunk 1]
```diff
@@ -23,6 +24,16 @@ public SmoothActionBarDrawerToggle(Activity activity, DrawerLayout drawerLayout,
     }
 
     @Override
+    public void onDrawerOpened(View drawerView) {
```
Comment (the-dagger) [inline android/app/src/main/java/org/fossasia/openevent/utils/SmoothActionBarDrawerToggle.java]: Umm, why are you adding these methods, if you are just calling super inside them?
I can't see these methods being overridden in the code that we already have.

**Comment by arp95** [comment]
@the-dagger please have a look now.

**Comment by arp95** [comment]
@the-dagger 
give me some time. as i want @mariobehling to update me the device on which its causing this as many fixes are possible for this.

**Comment by arp95** [comment]
@the-dagger sorry for the delay!
now you can have a look.

**Comment by the-dagger** [comment]
@arp95 have you tested this on a kitkat device to see if this works over there?

**Comment by arp95** [comment]
@the-dagger 
yes have tested it on kit kat device.

screenshot: 
18338976_10211217210580558_1023225906_o

**Pull request merged**